nipype/interfaces/io.py

```diff
@@ -2863,3 +2864,52 @@ def _list_outputs(self):
 
     def _add_output_traits(self, base):
         return add_traits(base, list(self.inputs.output_query.keys()))
+
+
+class ExportFileInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='Input file name')
+    out_file = File(mandatory=True, desc='Output file name')
+    check_extension = traits.Bool(True, desc='Ensure that the input and output file extensions match')
+    clobber = traits.Bool(desc='Permit overwriting existing files')
+
+
+class ExportFileOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='Output file name')
+
+
+class ExportFile(SimpleInterface):
+    """Copy in_file to out_file.
+
+    This interface copies an input file to a named output file.
+    This is useful to save files to a specified location
+    (as opposed to using DataSink).
+
+    Examples
+    --------
+
+    A trivial example that copies temporary_file.nii.gz
+    to sub1_out.nii.gz. (A more realistic example would set
+    in_file as the output of another Node.)
+
+    >>> from nipype.interfaces.io import ExportFile
+    >>> import os.path as op
+    >>> ef = Node(ExportFile(), "export")
+    >>> ef.inputs.in_file = "temporary_file.nii.gz"
+    >>> ef.inputs.out_file = op.abspath("output_folder/sub1_out.nii.gz")
```

Comment:
Your interface doesn't make directories that don't exist, so you're going to either need to target an existing directory or `mkdir` one. You can just `os.mkdir('output_folder')` to continue using this example.